_templates/component/with-prompt/template.ejs.t

```diff
@@ -6,5 +6,8 @@ interface <%= h.inflection.transform(name, ['underscore', 'camelize']) %>Templat
 }
 
 export default (render: Function, data: <%= h.inflection.transform(name, ['underscore', 'camelize']) %>TemplateData) => {
-    return render`Fill me`;
+    return render`
+        <style>${require('./style').toString()}</style>
+        <div>Fill me</div>
```

Comment:
can we have the <div> using the external template file as we discussed it here: https://github.com/biotope/experimental-biotope-components/issues/20